src/PKSim.Infrastructure/ProjectConverter/v12/Converter11To12.cs

```diff
@@ -152,6 +161,26 @@ private void convertIndividual(Individual individual)
       private void convertSimulation(Simulation simulation)
       {
          convertIndividual(simulation.BuildingBlock<Individual>());
+
+         addMissingExpressionBuildingBlock(simulation);
+      }
+
+      private void addMissingExpressionBuildingBlock(Simulation simulation)
+      {
+         //in v11, the usage of the expression building block was not saved in the used building block. We need to add it if it's not there already
```

Comment:
This comment seems disconnected from the code. You might even want to place this comment on line 164.